src/jshint.js

```diff
@@ -102,6 +103,11 @@ var JSHINT = (function () {
     extraModules = [],
     emitter = new events.EventEmitter();
 
+  var experimental = new Experimental(addlabel, advance, blockstmt, doFunction, error,
+    expression, function () { return funct; }, function() { return inblock; },
+    isEndOfExpr, nobreaknonadjacent, nolinebreak, optionalidentifier, peek, prefix,
+    warning, warningAt);
```

Comment:
I know but how do you expose method within the closure otherwise? JSHINT need to use prototype, but cannot really write this on my own.